src/uquickviewpanel.pas

```diff
@@ -215,7 +215,14 @@ procedure TQuickViewPanel.FileViewChangeActiveFile(Sender: TFileView; const aFil
       else begin
         // We can use the file source directly.
         FFileSource := Sender.FileSource;
-        ActiveFile:= aFile.Clone;
+        if aFile.IsNameValid then begin
+           ActiveFile:= aFile.Clone;
+        end else begin
+          if FFileSource.IsPathAtRoot(FFileSource.GetParentDir(aFile.Path)) then
```

Comment:
Why this special condition used? Why not show `/` root directory as is? This code works fine:
```
        if aFile.IsNameValid then begin
           ActiveFile:= aFile.Clone;
        end else begin
          ActiveFile:= TFile.Create( FFileSource.GetParentDir(aFile.Path) );
        end;
```
Am I missing something?